packages/react/package.json

```diff
@@ -40,7 +40,7 @@
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.18.3",
-    "eslint-plugin-react-hooks": "^2.4.0",
+    "eslint-plugin-react-hooks": "4.2.0",
```

Comment:
Just wondering.. why did this change? It seems unrelated :thinking:

Reply:
The old version has an unmeet peer dependency for eslint, which makes the e2e with npm 7 to fail. Seems like we have already upgraded to the latest version.